From: Miroslav Benes <mbenes@suse.cz>
Date: Mon, 30 Mar 2015 15:47:40 +0200
Subject: kgr: add error code to the message in kgr_revert_replaced_funs
Patch-mainline: submitted for review
References: fate#313296

When the error message is printed the appropriate error code is
appended for the user's convenience. Add it also to pr_err in
kgr_revert_replaced_funs to be consistent.

Signed-off-by: Miroslav Benes <mbenes@suse.cz>
Reviewed-by: Petr Mladek <pmladek@suse.cz>
Signed-off-by: Jiri Slaby <jslaby@suse.cz>
---
 kernel/kgraft.c |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/kernel/kgraft.c
+++ b/kernel/kgraft.c
@@ -839,8 +839,8 @@ static int kgr_revert_replaced_funs(stru
 
 				ret = kgr_patch_code(pf, false, true, true);
 				if (ret < 0) {
-					pr_err("kgr: cannot revert function %s in patch %s\n",
-					      pf->name, p->name);
+					pr_err("kgr: cannot revert function %s in patch %s (%d)\n",
+					      pf->name, p->name, ret);
 					pf->loc_old = loc_old_temp;
 					kgr_patching_failed(p, pf, true);
 					return ret;
